README.rst

```diff
@@ -40,14 +40,57 @@ USAGE
 
 If you want to learn more about coala-bears, its functionality and its usage,
 please take a look at our
-`tutorial <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#what-is-a-bear>`__.
+`tutorial <http://coala.readthedocs.org/en/latest/index.html>`__.
 
 ABOUT
 =====
 
-coala-bears is a Python package containing all the bears that is used by coala.
-
-Read more at our `documentation <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#guide-to-write-a-bear>`__.
+coala-bears is a Python package containing all the bears that are officially
+supported by coala. It features more than **65 bears** covering **32 languages**.
+You can see all of them `here <https://github.com/coala-analyzer/coala/wiki/Bears>`_.
+
+The list of supported languages include such famous languages, as:
+
+- C++
+- C#
+- CMake
+- CoffeeScript
+- CSS
+- Dart
+- Go
+- Haskell
+- HTML
+- Java
+- JavaScript
+- Julia
+- Latex
+- Lua
+- Markdown
+- Matlab/Octave
+- Natural Language (English)
+- Perl
+- PHP
+- Python 2
+- Python 3
+- R
+- RST
+- Ruby
+- Scala
+- SCSS
+- sh & bash scripts
+- SQL
+- TypeScript
+- Vimscript
+- XML
+- YAML
```

Comment:
i also disagree with having a list here, nobody will look at it here, better create a wiki page and give a link to there. i can do that if you want, will post the link soon